ts-rs/src/export.rs

```diff
@@ -334,11 +334,19 @@ fn generate_imports<T: TS + ?Sized + 'static>(
 /// Returns the required import path for importing `import` from the file `from`
 fn import_path(from: &Path, import: &Path) -> Result<String, ExportError> {
     let rel_path = diff_paths(import, from.parent().unwrap())?;
-    let path = match rel_path.components().next() {
-        Some(Component::Normal(_)) => format!("./{}", rel_path.to_string_lossy()),
+    let str_path = match rel_path.components().next() {
+        Some(Component::Normal(_)) => {
+            format!("./{}", rel_path.to_string_lossy())
+        }
         _ => rel_path.to_string_lossy().into(),
     };
 
+    let path = if cfg!(target_os = "windows") {
+        str_path.replace('\\', "/")
```

Comment:
I dont think this is sufficient, since paths can contain \.

We might need to iterate over the path components, not sure though.

Reply:
> I dont think this is sufficient, since paths can contain \.

Wdym? Is a component allowed to contain `\`? This replaces every `\` for `/`

Reply:
It seems `\` is allowed in Unix file/directory names, but this shouldn't be a problem, as the replace only happens if the user's OS is Windows, where `\` is not allowed.

I also suspect TS itself would not be very fond of a file containaing `\` in its name.

It could even be a reasonable idea to throw a compile time error if `#[ts(export_to)]` or `TS_RS_EXPORT_DIR` contain backslashes as that would probably produce unintended behavior and destroy portability, by having a path that can't exist on Windows. Of course, doing so would be another of those "*technically* a breaking breaking change, but probably won't affect anyone" versions

Reply:
I think you're right, sorry about that!

I thought I remembered some weird rules for paths on windows, but i was mistaken.